packages/nextjs/app/builders/_components/MemberList.tsx

```diff
@@ -0,0 +1,94 @@
+import { useEffect, useState } from "react";
+import Link from "next/link";
+import { Address } from "~~/components/scaffold-eth";
+import { useScaffoldEventHistory } from "~~/hooks/scaffold-eth";
+
+interface ValidPages {
+  [key: string]: boolean;
+}
+
+export const MemberList = () => {
+  const {
+    data: events = [],
+    isLoading: isLoadingEvents,
+    error: errorReadingEvents,
+  } = useScaffoldEventHistory({
+    contractName: "BatchRegistry",
+    eventName: "CheckedIn",
+    fromBlock: 122221841n,
+    watch: true,
```

Comment:
Maybe we don't need watch? Check In seems like a slow activity and also will stop at some point, so maybe we could remove it, just so that we don't keep requesting events again and again (saving load on rpc server)